cloudevents/formats/json-format.md

```diff
@@ -315,6 +318,39 @@ content-type: application/json
 }
 ```
 
+Example event with [JSON Number][json-number]-valued `data` and a content type
+declaring JSON-formatted data:
+
+```JSON
+{
+    "specversion" : "1.0",
+    "type" : "com.example.someevent",
+    "source" : "/mycontext",
+    "subject": null,
+    "id" : "C234-1234-1234",
+    "time" : "2018-04-05T17:31:00Z",
+    "comexampleextension1" : "value",
+    "comexampleothervalue" : 5,
+    "datacontenttype" : "application/json",
+    "data" : 1.5
+}
+```
+
+The above example re-encoded using [HTTP Binary Content Mode][http-binary]:
+
+```
+ce-specversion: 1.0
+ce-type: com.example.someevent
+ce-source: /mycontext
+ce-id: C234-1234-1234
+ce-time: 2018-04-05T17:31:00Z
+ce-comexampleextension1: value
+ce-comexampleothervalue: 5
+content-type: application/json
+
+1.5
+```
+
```

Comment:
Should we also include a binary data example to go along with your new text to re-enforce the idea of not making assumptions?

Reply:
Could do - I'm not 100% sure it's worth it though. Let's discuss in the meeting later, and I can add it in if desired.

Reply:
Yes, it's merely my inefficiency that has failed here.

Reply:
Added a new example - we probably want to haggle over the wording a little.